include/pb_view.h

```diff
@@ -40,6 +40,8 @@ class pb_view {
 		std::string prepare_keymaphint(keymap_hint_entry * hints);
 		pb_controller * ctrl;
 		newsbeuter::stfl::form dllist_form;
+		friend class pb_controller; // to use newsbeuter::stfl::form::set()
```

Comment:
This gets the job done, but also opens up a lot of possibilities for problems in the future. The two classes might become very entangled and it'll be a real pain to split them back into separate parts.

Instead of this, I suggest adding a new public method to `pb_view`, akin to `view`'s `set_message`.